cylc/rose/stem.py

```diff
@@ -17,24 +17,20 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""rose stem [path]
+"""rose stem [path] [options]
 
-Install a rose-stem suite using cylc install.
+Rose stem:
+- Allows users to specify source trees to install using --source.
+- Provides Cylc install with standard jinja2 variables
+  These variables allow suite designers to group tasks for users to
+  specify with --group/--task.
 
-To run a rose-stem suite use "cylc play".
 
-Install a suitable suite with a specified set of source tree(s).
+Note: At Cylc 8, rose stem acts as a wrapper to Cylc install. You will
+need to use cylc play to start the workflow.
```

Comment:
I think this is now likely to be missed, the important thing to convey is:

* Install a rose-stem suite using cylc install.
* To run a rose-stem suite use "cylc play".

Reply:
I'm not sure we really need to say this if we make it clear above that this is just a wrapper to "cylc install".